js/views/modals/purchase/Receipt.js

```diff
@@ -0,0 +1,56 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseModal from '../BaseModal';
+import Order from '../../../models/purchase/Order';
+import Listing from '../../../models/listing/Listing';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+
+    if (!this.model || !(this.model instanceof Order)) {
+      throw new Error('Please provide an order model');
+    }
+
+    if (!options.listing || !(options.listing instanceof Listing)) {
+      throw new Error('Please provide a listing model');
+    }
+
+    this.listenTo(this.model.get('items').at(0), 'change', () => this.render());
+    this.listenTo(this.model.get('items').at(0).get('shipping'), 'change', () => this.render());
+  }
+
+  className() {
+    return 'receipt flexColRows gutterVSm';
+  }
+
+  get prices() {
+    // create an array of price objects that matches the items in the order
```

Comment:
This is an array because it's an array in the order model, to support future code that has multiple items in each order.